virttest/qemu_vm.py

```diff
@@ -2288,7 +2281,7 @@ def sort_key(dev):
                 if set_disable_modern == "yes":
                     add_disable_modern(devices, device, dev_type)
 
-        return devices
+        return devices, spice_options
```

Comment:
Actually the `spice_options` might not be defined so please initialize it on top and not in if-branch.

Reply:
yes you are right. I overlooked it. I will correct it.